packages/amplify_flutter_web/lib/interop/ConfigOptionsImpl.dart

```diff
@@ -1,24 +1,35 @@
 /*
  * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
  *
- * Licensed under the Apache License, Version 2.0 (the "License").
+ * Licensed under the Apache License, Version 2.0 (the License).
```

Comment:
I don't know where these changes came from. Maybe auto formatting. I can remove them if anyone would like. I think for POC its fine.